data/campaigns/Winds_of_Fate/_main.cfg

```diff
@@ -21,9 +21,9 @@
 
 " + _ "(Hard level, 11 scenarios.)"
 
-    {CAMPAIGN_DIFFICULTY EASY      "units/drakes/burner-fly-1.png~CROP(0,0,72,72)~RC(magenta>red)"        ( _ "Aspirant")  ( _ "Normal")}
-    {CAMPAIGN_DIFFICULTY NORMAL    "units/drakes/fire-fire-se-1.png~RC(magenta>red)"                      ( _ "Intendant") ( _ "Challenging")} {DEFAULT_DIFFICULTY}
-    {CAMPAIGN_DIFFICULTY HARD      "units/drakes/inferno-fire-se-3.png~RC(magenta>red)"                   ( _ "Dominant")  ( _ "Difficult")}
+    {CAMPAIGN_DIFFICULTY EASY      "units/drakes/burner-fly-1.png~CROP(0,0,72,72)~RC(magenta>red)"        ( _ "Aspirant")  ( _ "Easy")}
```

Comment:
I think the easier difficulties might still need to be a bit easier, given that you added the recall costs in here. New players make some surprising choices regarding gold management and exp, so giving them extra gold might be helpful.

Reply:
I can bring it lower if you wish, but (for scenarios where recruiting is possible) it is currently giving beginner players twice as much gold and income as their opponent team.

In the scenarios where villages are important `village_support=0` for the opponent team essentially means half income and in the final "eyrie scenarios" S10 and S11, players also get double `village_gold` and `village_support`.

Reply:
Okay let's let it be as you have it in the pr.